knowledge-base/setup-fiddler-alongside-outlook.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Setting Up Fiddler Everywhere alongside Outlook
+description: "Learn how to configure your proxy settings to bypass specific Microsoft endpoints so that you could use MS products like Outlook alongside Fiddler Everywhere."
+type: how-to
+slug: fiddler-and-outlook
+publish: true
+res_type: kb
+---
+
+## Environment
+
+|   |   |
+|---|---|
+| Fiddler Everywhere | 1.0.0 and above |
+| 3rd-party application | Microsoft Outlook, Office365 |
+| Operating System | Windows, macOS |
+
+## Description
+
+Some third-party applications like **Microsoft Outlook** are not working seamlessly alongside MITM proxies like Fiddler Everywhere.  How can I handle this issue?
+
+## Solution
+
+In some cases, the third-party implementation restricts the application usage when the traffic goes through the MITM proxy. In other cases, the applications are not handling well dynamic change of the system proxy. The desktop application **Outlook** is one example of an application that might stop working when Fiddler Everywhere acts as a system proxy. To resolve issues with Outlook not working when Fiddler is turned on, follow these steps:
+
+- Exit the **Microsoft Outlook** application. Make sure it is not running in the background. 
```

Comment:
- Don't use bold for product names.
- Exit the **Microsoft Outlook** application -> Close Microsoft Outlook.